hack/api-description-whitelist.txt

```diff
@@ -97,6 +97,7 @@ v1.identity
 v1.identitylist
 v1.image
 v1.imagechangetrigger
+v1.imagelayer
```

Comment:
We don't add descriptions for nested fields today, until @stevekuznetsov fixes our code to use Godoc descriptions for swagger, at which point all these become redundant.